projects/igniteui-angular/src/lib/dialog/dialog-content.component.html

```diff
@@ -7,7 +7,7 @@
         <ng-content *ngIf="!title" select="igx-dialog-title,[igxDialogTitle]"></ng-content>
 
         <div class="igx-dialog__window-content">
-            {{ message }}
+            <span *ngIf="message" class="igx-dialog__window-message">{{ message }}</span>
             <ng-content *ngIf="!message"></ng-content>
```

Comment:
I think you can still leave the `<ng-content></ng-content>` be rendered even if there's a provided message. There could be a use case with elements going below a message which is programmatically populated!

Reply:
I was thinking the same thing, but decide not to do it, because the message will have that max-width of 40ch which was the problem for the templated area, and if we have both, they will look misaligned, broken. The whole idea is to keep the current behavior of the dialog if it's not templated.

Reply:
@desig9stein Alright! I'm perfectly fine with this decision. I know that we're keeping the `[message]` input just for backwards compatibility anyways, so the second example that I gave would be the way to achieve the scenario that I've described. Thanks!